_config.yml

```diff
@@ -95,6 +95,7 @@ JB :
       colorscheme: light
    
   # Settings for analytics helper
+  # Set 'site.safe' to true to enable analytics plugin
```

Comment:
@themez we've moved away from using site.safe to decide if the site is in production mode. I'm killing this line and merging the rest.